editfonts/objects/bezierpen.py

```diff
@@ -0,0 +1,116 @@
+import math
+
+from gi.repository import Gdk
+
+from defcon import Contour
+from defcon import Point
+from editfonts.globals import globals
+
+
+def distance(x1, y1, x2, y2):
+    t1 = (x1 - x2)
+    t2 = (y1 - y2)
+    return math.sqrt(t1 * t1 + t2 * t2)
+
+class BezierPenTool(object):
+
+    def __init__(self, editorBox):
+        self.editor = editorBox
+        self.set_active(True)
+
+    def set_active(self, state):
+        if state == True:
+            self.is_active = True
+            self.contour = Contour()
+            self.contour.dirty = True
+            self.editor.add_contour(self.contour)
+            self.connect_editor()
+        else:
+            self.contour.dirty = False
+            self.is_active = False
+            self.disconnect_editor() 
+
+        globals.TOOL_ACTIVE['BezierPenTool'] = state
+
+    def get_active(self):
+        return self.is_active
+
+    def disconnect_editor(self):
+        self.editor.disconnect(self.handle_press)
+        #self.editor.disconnect(self.handle_release)
+
+    def connect_editor(self):
+        # add all the events here which the pen tool needs to listen to 
+        self.handle_press = self.editor.connect("button-press-event", self._on_point_press)
+        #self.handle_release = self.editor.connect("button-release-event", self._on_point_release)
+
+    def _on_point_press(self, widget, event):
+        if event.type == Gdk.EventType.BUTTON_PRESS\
+                 and event.button == 1:
+            print "Clicked on: (" + str(event.x) + ", " + str(event.y) + ")"    
```

Comment:
this line shows the following error: ./editfonts/objects/bezierpen.py:53:34: E901 SyntaxError: invalid syntax
How should I fix this?